extras/docker/docker-entrypoint.sh

```diff
@@ -33,10 +33,12 @@ file_env 'IDM_DB_USER'
 file_env 'IDM_ADMIN_ID'
 file_env 'IDM_ADMIN_USER'
 file_env 'IDM_ADMIN_EMAIL'
-file_env 'IDM_ADMIN_PASS' 
+file_env 'IDM_ADMIN_PASS'
 file_env 'IDM_EX_AUTH_DB_USER'
 file_env 'IDM_EX_AUTH_DB_PASS'
+file_env 'DATABASE_HOST'
```

Comment:
this parameter already exists, but was missing in the file !

Reply:
In that case I'm guessing that this line  adding `DATABASE_HOST` should be removed. Not all `ENV` variables are listed here, only **sensitive** ones. Maybe a comment line needs to be added above the existing eight variables describing which ones should be listed.

If it wasn't previously in the `Dockerfile`, then it wasn't considered **sensitive** and therefore doesn't need to have an alternate for protected by [Docker Secrets](https://docs.docker.com/engine/swarm/secrets/) - in other words it isn't a username or password that shouldn't be passed in plain text.  This pattern is also used by [MySQL](https://mysqlrelease.com/2017/08/docker-secrets-and-mysql-password-management/)

For example you don't want to do this:

```console
docker run -it fiware-idm -e IDM_EX_AUTH_DB_PASS=password
```

Because you can inspect the `ENV` variable and access the password. (in plain text)

But you can do this:

```console
docker run -it fiware-idm -e IDM_EX_AUTH_DB_PASS_FILE=./my_secret --secret my_secret_data
```

Its the function adding in that `_FILE` suffix and looking for the secret in the right place that is only available for a subset of configuration options.